lib/cli/cliPlugin.js

```diff
@@ -0,0 +1,346 @@
+'use strict';
+const CLI = require('./messages.js');
+const CLICommand = require('./cliCommand.js');
+const { getObjectFrom } = require('./cliTools');
+const tools = require('../tools.js');
+const fs = require('fs');
+const path = require('path');
+
+const controllerIoPackPath = path.join(__dirname, '../../io-package.json');
+
+/** Command iobroker plugin <enable/disable/status> <pluginname> [--host this/hostname] ... */
+module.exports = class CLIPlugin extends CLICommand {
+    /** @param {import('./cliCommand').CLICommandOptions} options */
+    constructor(options) {
+        super(options);
+    }
+
+    /**
+     * Executes a command
+     * @param {any[]} args
+     */
+    execute(args) {
+        const { callback, showHelp } = this.options;
+        const command = args[0];
+
+        switch (command) {
+            case 'on':
+            case 'enable':
+                return this.set(args, true);
+            case 'off':
+            case 'disable':
+                return this.set(args, false);
+            case 'status':
+                return this.status(args);
+            default:
+                CLI.error.unknownCommand('plugin', command);
+                showHelp();
+                return void callback(3);
+        }
+    }
+
+    /**
+     * Enables or disables the given plugin
+     * @param {any[]} args
+     * @param {boolean} enabled
+     */
+    set(args, enabled) {
+        const { callback, dbConnect } = this.options;
+        /** @type {string} */
+        const pluginName = args[1];
+        if (!pluginName) {
+            CLI.error.requiredArgumentMissing(
+                'pluginName',
+                'plugin enable <pluginname>'
+            );
+            return void callback(34);
+        }
+
+        /** @type {string | undefined} */
+        let hostname;
+        /** @type {string | undefined} */
+        let instance;
+
+        if (this.options.adapter || this.options.instance) {
+            instance = this.options.adapter || this.options.instance;
```

Comment:
so we also allow --adapter name.nr even if undocumented?

Reply:
... and name only and add .0 ... ok

Reply:
> so we also allow --adapter name.nr even if undocumented?

Yes, I wanted to anticipate people confusing instances with adapters.

Reply:
I updated the help output.